checker/bin-devel/test-cftests-junit.sh

```diff
@@ -13,3 +13,6 @@ source "$SCRIPTDIR"/clone-related.sh
 # Adding --max-workers=1 to avoid random failures in Github Actions. An alternative solution is to use --no-build-cache.
 # https://github.com/eisop/checker-framework/issues/849
 ./gradlew test -x javadoc -x allJavadoc --console=plain --warning-mode=all --max-workers=1
+
+# Test clean task
+./gradlew clean
```

Comment:
```suggestion
./gradlew clean
./gradlew clean
```

You need to run it twice, here and in the other tasks, to actually see what happens when you run `clean` on a `clean` directory. Otherwise, you again don't test what happens then.
The change just to `issue265/Makefile` is not enough.

This also doesn't really test that the directory is clean afterwards... but checking `git status` doesn't help, as most temporary files are being ignored.

Reply:
Ah, the change to `issue265/Makefile` is enough on CI. Let's run clean twice only here.

In `docs/examples/BazelExample/Makefile` can you change `bazelisk clean` to `-bazelisk clean` so that bazelisk isn't required for cleaning?

Reply:
Added!